cmd/cacherc/cmd/all.go

```diff
@@ -4,6 +4,7 @@ package cmd
 
 import (
 	"context"
+	"errors"
```

Comment:
pkg/errors has an `Is` func, is there a reason why you didn't go with that? I can only imagine it being even more awkward to mix errors and pkg/errors uses

Reply:
I looked for one, but it didn't exist in the version of `pkg/errors` used by cacher. I've upgraded it and addressed this awkwardness. Thanks!